src/main/java/httpServer/Messages.java

```diff
@@ -0,0 +1,27 @@
+package httpServer;
+
+public class Messages {
+    static String serverStartedMessage() {
```

Comment:
This would be a good use case for a `Message` enum:

```
enum Message {
  SERVER_STARTED("Server started"),
  CLIENT_CONNECTED("Client connected");

  private final String message;

  Message(String message) {
    this.message = message;
  }

  public String toString() {
    return this.message;
  }
}

Reply:
That's a great idea. I'll add this. Thanks, Matt